x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/security/authc/TokenAuthIntegTests.java

```diff
@@ -865,7 +659,7 @@ public void testCreatorRealmCaptureWillWorkWithClientRunAs() throws IOException
         // Create a token with client credentials and run-as, the token should be owned by the run-as user (native realm)
         var createTokenRequest1 = new org.elasticsearch.xpack.core.security.action.token.CreateTokenRequest();
         createTokenRequest1.setGrantType("client_credentials");
-        final PlainActionFuture<org.elasticsearch.xpack.core.security.action.token.CreateTokenResponse> future1 = new PlainActionFuture<>();
+        final PlainActionFuture<CreateTokenResponse> future1 = new PlainActionFuture<>();
         runAsClient.execute(CreateTokenAction.INSTANCE, createTokenRequest1, future1);
```

Comment:
This method should be rewritten to use the new convenient methods from TestSecurityClient. I can follow up with a separate PR (since I wrote the original version).